src/config/options.rs

```diff
@@ -49,6 +49,19 @@ pub enum ControlBraceStyle {
     AlwaysNextLine,
 }
 
+#[config_type]
+/// How to treat the semicolon that is optional for final diverging expressions
+/// (`return`/`break`/`continue`).
+pub enum TrailingSemicolon {
+    /// Always rewrite `return;` and `break;` expressions to have a trailing semicolon,
+    /// unless the block is a single-line block, e.g. `let PAT = e else { return }`.
+    Always,
+    /// Always return `return` and `break` expressions to remove the trailing semicolon.
```

Comment:
Just want to avoid any confusion with the docs. Since this is the `Never` case it might be better to not use the word "Always" in the description. One potential rewording:

```rust
/// Never add a trailing semicolon. This removes trailing semicolon's from `return` and `break` expressions.
```
Also, this won't remove them from `continue;`?